head.ts

```diff
@@ -1,13 +1,18 @@
-import { nth } from './nth.ts'
-
-/**
- * Returns the first element of the given list or string. In some libraries
- * this function is named `first`.
- *
- *      Fae.head(['fi', 'fo', 'fum']); //=> 'fi'
- *      Fae.head([]); //=> undefined
- *
- *      Fae.head('abc'); //=> 'a'
- *      Fae.head(''); //=> ''
- */
-export const head = nth(0)
+import { nth } from './nth.ts'
+import curryN from './utils/curry_n.ts'
+import { InferElementType } from "./utils/types.ts"

```

Comment:
It will be `import type { InferElementType } from './utils/types.ts'`
Also, take care double and single quotes. Use single quotes wherever possible.